modules/feincms-pages/scripts/init.js

```diff
@@ -7,8 +7,7 @@
         'feincms-pages.page-group',
         'feincms-pages.load-page',
         'feincms-pages.constants',
-        'feincms-pages.routes',
-        'feincms-pages.templates'
+        'feincms-pages.routes'
```

Comment:
Script sourcing is separate from module dependency registration. You can source `templates.js`, but if `feincms-pages.templates` is never registered as a dependency in the tree of dependencies of the bootstrapped module – usually `app` – then the templates will never be put in the `$templateCache`.

Given this setup:
```js
angular
    .module('app.directives', [])
    .run([
        function () {
            console.log('directives run');
        }
    ]);

angular
    .module('app.templates', [])
    .run([
        function () {
            console.log('templates run');
        }
    ]);
```

Try the following: only in the second case does is `app.templates` module loaded, even though the javascript that made it is always run in the setup above.

1.
```js
angular.module('app', [
    'app.directives'
]);
angular.bootstrap(document, ['app']);
// 'directives run'
```

2.
```js
angular.module('app', [
    'app.directives',
    'app.templates'
]);
angular.bootstrap(document, ['app']);
// 'directives run'
// 'templates run'
```